Plugins/DeleteUnusedBranches/DeleteUnusedBranchesForm.cs

```diff
@@ -163,18 +163,17 @@ private void Delete_Click(object sender, EventArgs e)
             {
                 await TaskScheduler.Default.SwitchTo(alwaysYield: true);
 
-                if (remoteBranches.Count > 0)
+                foreach (var remoteBranch in remoteBranches)
```

Comment:
This is probably a separate PR. Any issue?

Reply:
Agree. Needed this for testing. Issue was->1 branch could not be deleted for some reason. This resulted in none of the 1500 branches I wanted to delete was deleted. Didn't recieve an error, didn't know why it broke. This has nothing to do with this pull request though..:)